pkg/config/v3/config_test.go

```diff
@@ -307,6 +307,25 @@ var _ = Describe("cfg", func() {
 			Expect(c.HasGroup("other-group")).To(BeFalse())
 		})
 
+		It("ListResourceswithGK should be empty since there are no resources with specified GK", func() {
```

Comment:
Same, would you mind moving these tests to follow the same order than the interface?